client/gulpfile.js

```diff
@@ -67,6 +103,7 @@ gulp.task('watch', () => {
   })
   gulp.watch(sources.css.all, ['styles', 'stylelint'])
   gulp.watch(sources.scripts.all, ['scripts'])
+  gulp.watch(sources.images.svg.all, ['svgstore'])
```

Comment:
My thinking was that the asset directory paths would be mirrored by this structure (i.e. I think the svg images should live inside assets/images/svg). This will only reload the svg-sprite creation task when there are changes to .svg files in the assets/images/svg directory.